net-analyzer/luigi: EAPI bump and PV bump

xmhd opened the pull request:
as per title.

ebuild could still do with some further improvement, e.g. conf file and such.

varilci reviewed (changes requested):
You left ebuild of `luigi-3.0.2` untouched but deleted it from Manifest, hence pkgcheck is failing.

xmhd commented:
> You left ebuild of `luigi-3.0.2` untouched but deleted it from Manifest, hence pkgcheck is failing.

fixed fixed fixed

also added missing RDEPEND

varilci reviewed (approved):
looks good now
